doc/quickstart.md

```diff
@@ -0,0 +1,139 @@
+# The Quick Start Guide to Electrum Rust Server
+
+### Prepare bitcoind
+
+In your bitcoin.conf (usually sitting in /home/username/.bitcoin) ensure you have the following:
+```
+rpcallowip=127.0.0.1
+rpcallowip=10.0.0.0/8
+rpcallowip=172.0.0.0/8
+rpcallowip=192.0.0.0/8
+rpcuser=bitcoin
+rpcpassword=bitcoin
```

Comment:
Personally, I prefer using bitcoind cookie file, instead of hard-coding the password.
WDYT?

Reply:
I use the rpcuser/rpcpassword because other projects connect to bitcoind via user/pass, not cookie file.

Reply:
It'd be best to persuade other projects to use cookie. Which are missing?

Reply:
dojo, mempool.space